Dockerfile

```diff
@@ -3,12 +3,8 @@ FROM quay.io/centos/centos:stream9
 # Help people find the actual baremetal command
 COPY scripts/openstack /usr/bin/openstack
 
-RUN dnf install -y python3 python3-requests && \
-    curl https://raw.githubusercontent.com/openstack/tripleo-repos/master/plugins/module_utils/tripleo_repos/main.py | python3 - -b master current-tripleo && \
-    dnf update -y --setopt=install_weak_deps=False && \
-    dnf install -y --setopt=install_weak_deps=False python3-ironicclient python3-ironic-inspector-client && \
-    dnf clean all && \
-    rm -rf /var/cache/{yum,dnf}/* && \
+RUN dnf install -y python3 python3-requests python3-pip && \
```

Comment:
You don't need python3-requests any more, it was required for the tripleo-repos tool.